src/ts/components/transaction/Confirm.tsx

```diff
@@ -77,105 +77,122 @@ export default class Confirm extends React.Component<IConfirmProps, IConfirmStat
+          <Tooltip
+            title={!this.state.message ? t('copyToClipboard') : t('copiedExclam')}
+            position='bottom'
+            trigger='mouseenter'
+            arrow={true}
           >
-            Confirm
-          </ConfirmButton>
-        </Info>
-      </Section>
-    </Modal>
+            <FromAddress>
+              <span onClick={() => this.copyToClipboard(this.props.fromAddress)}>
+                {this.props.fromAddress.substring(0, 8) + '...' + this.props.fromAddress.substring(this.props.fromAddress.length - 10)}
+              </span>
+            </FromAddress>
+          </Tooltip>
+          <Heading>Confirm Extrinsic</Heading>
+          <Subheading>Review your extrinsic details</Subheading>
+        </UpperSection>
+        <OverlaySection>
+          <Overlay/>
+        </OverlaySection>
+        <Section>
+          <FromTo color={colorSchemes[this.props.color].backgroundColor}>
+            <Icon name='arrow circle right' size={'big'} style={{  'marginLeft': '10px' }}/>
+            <Container textAlign={'left'} style={{ 'marginLeft': '10px' }}>
+              <div style={{ 'display': 'flex', 'alignItems': 'center', 'fontSize': '13px', 'marginLeft': '-10px' }}>
+                <Identicon account={this.props.toAddress} size={20} style={{ 'marginRight': '5px' }}/>
+                <Tooltip
+                  title={!this.state.message ? t('copyToClipboard') : t('copiedExclam')}
+                  position='bottom'
+                  trigger='mouseenter'
+                  arrow={true}
+                >
+                  <span onClick={() => this.copyToClipboard(this.props.toAddress)}>{this.truncate(this.props.toAddress)}</span>
+                </Tooltip>
+              </div>
+              <div style={{ 'display': 'flex', 'alignItems': 'center', 'fontSize': '11px', 'marginLeft': '10px' }}>
+                <p>Available: {formatBalance(this.props.recipientAvailable)}</p>
+              </div>
+            </Container>
+          </FromTo>
+        </Section>
+        <Section style={{ 'marginTop': '8px' }}>
+          <Info>
+            <Key>Fee</Key>
+            <Value>{formatBalance(this.props.fee)}</Value>
+          </Info>
+          <div style={{ 'border': '1px solid gray' }}/>
+        </Section>
+        <Section style={{ 'marginTop': '8px' }}>
+          <Info>
+            <Key>Amount</Key>
+            <Value>{formatBalance(this.props.amount)}</Value>
+          </Info>
+        </Section>
+        <Section style={{ 'marginTop': '8px','marginBottom': '16px' }}>
+          <Info>
+            <Key>Total</Key>
+            <Value>{formatBalance(this.props.amount.add(this.props.fee))}</Value>
+          </Info>
+        </Section>
+        {warning}
+        <Section>
+          <Info>
+            <div style={{ 'fontSize': '11px' }}>
+              <div>
+                <Icon name='arrow right' size={'small'}/>
+                Fees include the transaction fee and the per-byte fee
```

Comment:
i think we need to move all these text to translate service later. but let's get a working one first.